src/main/java/org/opensearch/plugin/insights/rules/transport/top_queries/TransportTopQueriesAction.java

```diff
@@ -86,6 +86,16 @@ protected TopQueriesResponse newResponse(
             default:
                 size = clusterService.getClusterSettings().get(QueryInsightsSettings.TOP_N_LATENCY_QUERIES_SIZE);
         }
+        final String[] timeRange = topQueriesRequest.getTimeRange();
+        if (timeRange[0] != null && timeRange[1] != null) {
```

Comment:
Let's check timeRange length==2 before this, otherwise there are possibilities this will cause unexpected exceptions in the future

Reply:
I'll leave it as checking the length == 2 for now, but feel free to change the getting of from and to!